plugin/restore/mysql/mysql.go

```diff
@@ -348,31 +456,36 @@ func (r *Restore) SyncArchivedBinlogFiles(ctx context.Context, instance *api.Ins
 		if _, ok := todo[serverFile.Name]; !ok {
 			continue
 		}
-		if err := r.downloadBinlogFile(ctx, instance, binlogDir, serverFile); err != nil {
+		if err := r.downloadBinlogFile(ctx, serverFile); err != nil {
 			return fmt.Errorf("cannot sync binlog %s, error: %w", serverFile.Name, err)
 		}
 	}
 
 	return nil
 }
 
-// SyncLatestBinlog syncs the latest binlog between the instance and `binlogDir`
-func (r *Restore) SyncLatestBinlog(ctx context.Context, instance *api.Instance, binlogDir string) error {
+// fetchBinlogUpToTargetTs fetches the binlog from server up to the targetTs.
+// TODO(zp): It is not yet supported to synchronize some binlogs.
+// The current practice is to download all binlogs, but in the future we hope to only synchronize to the PITR time point
+func (r *Restore) fetchBinlogUpToTargetTs(ctx context.Context) error {
```

Comment:
where is targetTs, if it's not supported now, then we can call it incrementalFetchAllBinlogFiles